src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+
+(set: $dissolve to (transition: 'dissolve') + (transition-time: 3s))
+(set: $hatchet to true)(set: $multitool to true)
+(set: $oldSchoolJuggalosSeen to false)(set: $jarSeen to false)(set: $justLeftTheGutter to false)(set: $golfCartSeen to false)(set: $cartPartsSeen to 0)(set: $wheelsSeen to false)(set: $engineSeen to false)(set: $defenseSeen to false)(set: $hornSeen to false)(set: $gutterStatus to "intro")
+
+::The Gutter - Outside hub
+(if: $justLeftTheGutter is true)[(set: $justLeftTheGutter to false)I walked back into the dim neon light of the Gutter’s sign. I could see:]
+\(else:)[In the dim light of the neon sign, I could see:]
+
+>>[[The Gutter’s beaten-to-hell block of a front door|The Gutter - Inside $gutterStatus]]
+>>[[The paved path to The Church|The Church - intro]]
+>>[[The dirt road to the Roach Market|RM-$roachEntrance-entrance]]
+(if: $oldSchoolJuggalosSeen is false)[>>[[Some old-school juggalos in a heated conversation by the side of the building|The Gutter - Outside - Old-school juggalos]]]
+(if: $golfCartSeen is false)[>>[[Ron’s shitheap golf cart|The Gutter - Golf cart]]]
+(if: $jarSeen is false)[>>[[The jar|The Gutter - Jar]]]
+
+::The Gutter - Outside - Old-school juggalos
+A tall, chubby juggalo stroking a long beard and a shorter, musclebound ninja with green Kool-Aid dyed hair were inches away from each other, shouting in agreement as they chainsmoked:
 
-I picked [[the least muddy trail towards civilization|The Gutter - Outside - Approach]].
+“Sorry to be all deep and serious, but I got a lotta shit on my mind.”
 
-::The Gutter - Outside - Approach <1390,73>
-My body and mind were screaming as [one]<trigger1|.
+“It breaks my heart dude, but it had to be said. Ninjas today just don’t know.”
 
-|section2)[I kept my head down and my hood up. I used all of my energy to move forward through the tarp-covered alleyways of Gatherville.
+“They don’t, man! The Dark Carnival saved my life, dude, and it’s about love. Shit is about family, dude. If these ninjas were out there listening to the shit...”
 
-My gut lurched as I realized I’d need to ask around for more information on Anna. The last thing I wanted to do was [talk to anyone]<trigger2|.]
+“Listen to the //shit//!!”
 
-|section3)[My brain bounced into the darkness of reality, searching for answers. It was still pretty early in the night. Around this time, anyone who knew anything was
+(click: "Listen to the shit")[$dissolve[The shorter ninja started jumping and kicking an unknown assailant at this point. Large picked up on the energy and took his hands away from his beard. He started punching one hand into the other to punctuate his points.
 
-[[>>Getting fucked up at the Gutter|The Gutter - Outside - Intro]]
-[[>>Looking to score at the Roach Market|Market scene]]
-[[>>Catching an early show at the Church|Church scene]]]
+“Exactly! If they were listening; like Dark Lotus? '//Juggalo Family//'? That’s my real shit. That’s what I’m about.”
```

Comment:
Song titles should be in double quotes and not italicized, per MLA/AP/etc.

Reply:
Also, I know I'm a little gross with my em-dash usage, but I feel like an em-dash fits these two characters better than that semicolon — and it fits the pacing of their enthusiastic conversation, too, I think.

Reply:
Dammit! I always swap the MLA style for albums and song titles. Removed the italics, but keeping the single quotes (since this is already in a quoted section of text).

I agree on the em-dash.